invenio_circulation/config.py

```diff
@@ -32,7 +32,8 @@
 CIRCULATION_DOCUMENT_RETRIEVER_FROM_ITEM = None
 """Function that returns the Document PID of a given Item PID."""
 
-CIRCULATION_STATES_ITEM_AVAILABLE = ['ITEM_RETURNED', 'CANCELLED']
+CIRCULATION_STATES_ITEM_AVAILABLE = ['CREATED', 'PENDING', 'ITEM_RETURNED',
```

Comment:
I thought we discussed to invert this logic to define the states that an item is considered on loan and then refactor the logic in the code. Do I miss something?